### PR TITLE
Fixed bugs when updating app pools

### PR DESCRIPTION
**Pull request by jimmyca15**
This PR fixes problems described in [this issue](https://github.com/Microsoft/IIS.Administration/issues/190)

1. The application pool identity password was not being sent to the server when the field was updated. This was due to a bug in how changes were detected.
2. The Application pool recycling switch contained an unfriendly behavior where it would close if invalid values were entered. A change was made to allow values to be rectified if entered incorrectly.

### CODE REVIEW

**Comment by jimmyca15** [comment]
Due to the ongoing development taking place in the dev branch. This hotfix PR is targeting the hotfix branch. The hotfix branch will merge into master for a hotfix release.

File: app/webserver/app-pools/recycling.component.ts [hunk 1]
```diff
@@ -228,6 +228,6 @@ export class RecyclingComponent {
     }
 
     timeIntervalEnabled(): boolean {
-        return this.model.periodic_restart.time_interval < TimeSpan.MaxMinutes;
+        return this.model.periodic_restart.time_interval != TimeSpan.MaxMinutes;
```
Comment (shhsu) [inline app/webserver/app-pools/recycling.component.ts:231]: Need some clarifications here.
Is 432000 effectively the recycling timespan if timespan is disabled?

Comment (jimmyca15) [reply app/webserver/app-pools/recycling.component.ts:231]: Yes.

**Comment by shhsu** [comment]
🚢

**Review by shhsu — APPROVED** [review]

**Pull request merged**